### ISSUE TITLE
V 2.4.1 on iOS - Notifications/Alerts not happening

### ISSUE THREAD

**Issue by gary-menzel** [Manuito83/torn-pda#145]
I can see in the releases that 2.4.0 is the latest release, except my About screen says 2.4.1.  Both on an iPhone and an iPad with latest iOS (although it was happening before the upgrade to the current iOS version).

The issue is that notifications/alerts are not being triggered despite having turned them on and off in TornPDA, and also checked that Notifications are enabled in the Settings page of the device.

I have also reset server notifications from within the app.

Getting to the end of the things I can do so reporting that I am unable to get notifications.

**Comment by Manuito83** [comment]
Thanks. Would you mind contacting me in game or Discord so that I can try to troubleshoot it?

**Comment by gary-menzel** [comment]
The reset on the iPad now seems to be working (although notifications from the the iPhone are more useful due to linking to the watch).

I'm going to try something else and I will let you know if there is no resolution.

Thanks for you prompt reply.

**Comment by gary-menzel** [comment]
Thanks again....

It appears that either a RESET from the iPad version of TornPDA (or a full shutdown/startup on the iPhone) has probably (I did get two alerts through from the iPhone) resolved the issue.  Although, it is strange because iOS was only updated (on both devices) yesterday and both devices did a restart.

I'll close this now - sorry to take up your time.

**Comment by Manuito83** [comment]
No worries! Let me know if it fails again, there a couple of things we can try. Cheers!

**Comment by gary-menzel** [comment]
I thought this had been resolved however....

The iPad seems to be reporting the notifications - including stock refills - and they appear on the home screen as would be expected.

The iPhone seems to only seems to produce notifications when TornPDA is running,  but only specific ones like Life or Energy refill and never an stock refills.

I have joined your discord server (and I can see you are currently offline in the users list) - let me know which way is best to see if I can get the notifications working properly on the iPhone.

**Comment by gary-menzel** [comment]
Thanks for you help on sorting the notifications out.

As a result of our discussion, the first thing I tried was just removing my API Key and reentering it.  However, it appears the App retained the configuration from before (all my settings were there).

Then I removed the key again, deleted the app and downloaded it again from the store and set everything back up.

I have just received a foreign stock alert on the phone and watch.  So I think it is now resolved.

The process of deleting the app and reinstalling it seems to have cleared whatever the issue was.  The reinstall is not so bad - just clicking a bunch of checkboxes to get things back as I want them - and also downloading my targets from YATA again.

Once again, thanks for your help.
